src/core/counter/mod.rs

```diff
@@ -303,6 +304,16 @@ impl Counter {
 
         Ok(())
     }
+
+    pub fn qb64b_size(&self) -> Result<u32> {
+        let sizes = sizage(&self.code)?;
+        Ok(sizes.fs)
+    }
+
+    pub fn qb2_size(&self) -> Result<u32> {
+        let sizes = sizage(&self.code)?;
+        Ok(sizes.fs * 3 / 4)
+    }
```

Comment:
Why do we need the two? Did this come from KERIpy?

Reply:
Here is the corresponding keripy code: https://github.com/WebOfTrust/keripy/blob/development/src/keri/core/coring.py#L4349 
`qb64b` and `qb2` sizes differs there

Reply:
That's just the instantiation code that can parse different inputs - I don't see a rawSize for Counter and I don't think it makes sense, since a `Counter` is just a code and a count, there is no raw material so `raw_size()` should always be 0. If you base your computation on full size alone you are including the code, which is what raw size typically excludes. I think you should remove the changes in this file.

Reply:
It's not called `rawSize` here (`qb2_size` , `qb64b_size`). 
In python they cut input bytes right in the Counter constructor.
In our case  parside cutting stream and we need some method  to get counter size.

Reply:
Ahh I see what you are saying. Thanks for clarifying.

Reply:
If what you really need is full size for stripping, maybe the other methods need adjustment?

Reply:
Can't you just strip after you parse, and use qb64().len() or qb2().len()?

Reply:
As Sam said: each Cesr type is self-descriptive and parser always knows the length. I think it's a most correct way to provide it. It's legal to use the sizage math in parser.